api/server/sdk/volume_node_ops.go

```diff
@@ -151,33 +151,44 @@ func (s *VolumeServer) Mount(
 	}
 
 	// Get access rights
-	if err := s.checkAccessForVolumeId(ctx, req.GetVolumeId()); err != nil {
-		return nil, err
-	}
-
-	vols, err := s.driver().Inspect([]string{req.VolumeId})
+	// Checks for ownership
```

Comment:
Are we only checking for ownership now? And this is done implicitly via an inspect? Instead of the additional `s.checkAccessForVolumeId`

Reply:
checkAccessForVolumeId calls inspect for access, but now this call needs to call inspect directly to get access to the response and get the Volume object.